iothub/device/src/Transport/Amqp/AmqpLinkHelper.cs

```diff
@@ -0,0 +1,325 @@
+﻿using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Framing;
+using Microsoft.Azure.Devices.Shared;
+using System;
+using System.Globalization;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Client.Transport.Amqp
+{
+    internal class AmqpLinkHelper
```

Comment:
>AmqpLinkHelper [](start = 19, length = 14)

Create interface for testablity

Reply:
This class contains all static functions, not suitable for interface.